convars.cfg

```diff
@@ -0,0 +1,9 @@
+# Here you can easily configure the different requirements for various resources
+
+# POLICE REQUIREMENTS
+setr storerob_minimumpolice 1 # Minimum number of police required to rob a store
```

Comment:
IMO. 
We should handle config consistently.
This method with convar's is very repetitive due to not supporting tables.